### PR TITLE
LPS-75482 Do not insert feedback spinner icon when container has form field options

### PR DESCRIPTION
**Pull request by alinedoleron**

### CODE REVIEW

**Comment by alinedoleron** [comment]
ci:test

**Comment by alinedoleron** [comment]
ci:test

File: dynamic-data-mapping-form-renderer/src/main/resources/META-INF/resources/js/field_feedback_support.js [hunk 1]
```diff
@@ -145,17 +145,18 @@ AUI.add(
...
-				instance.getInputNode().insert(
-					Lang.sub(
-						TPL_FEEDBACK,
-						{
-							icon: icon
-						}
-					),
-					'after'
-				);
+				if (!container.get('children').hasClass('liferay-ddm-form-field-options')[0]) {
```
Comment (leoadb) [inline dynamic-data-mapping-form-renderer/src/main/resources/META-INF/resources/js/field_feedback_support.js]: @alinedoleron It looks strange the way we are doing this condition.

**Comment by alinedoleron** [comment]
ci:test

**Comment by brunobasto** [comment]
@alinedoleron, we should try to fix it another way. field_feedback_support.js is inside from-renderer, and it should not have code related to specific fields inside it.

Can you hide it using css specific for the options-field?

**Comment by alinedoleron** [comment]
oh, ok @brunobasto. I'll try to fix with css. Thank you.

**Comment by alinedoleron** [comment]
Closed PR
This new PR was sent https://github.com/liferay/com-liferay-dynamic-data-mapping/pull/628

**Pull request closed without merging**